PxAPI-2.yml

```diff
@@ -310,40 +310,43 @@ components:
             discontinued:
               type: boolean
               nullable: true
-    Paginated:
+
+    PageInfo:
       type: object
       properties:
-        page:
-          description: Pagination properties
-          title: PaginationProperties
-          type: object
-          properties:
-             pageNumber:
-              type: integer
-              format: int32
-             pageSize:
-              type: integer
-              format: int32
-              example: 33
-             totalElements:
-              type: integer
-              format: int32
-             totalPages:
-              type: integer
-              format: int32
+        pageNumber:
+          type: integer
+          format: int32
+          minimum: 1
+        pageSize:
+          type: integer
+          format: int32
+          example: 20
+          minimum: 5
+          maximum: 500
+        totalElements:

```

Comment:
max/min ?

Reply:
That you can specify the page size to between 5 and 500. Although I am not sure the upper constrain of 500 is the best choice.

Reply:
min=1 and max unbounded seems better ?

Reply:
Maybe, this would practically mean that a user will be able to fetch all tables in one request which could be good in some cases and in others it might be misused to degrade the responsiveness of the service.

Reply:
Yes, misuse is a consern. I think there has to be a maxvalue at runtime, but it should come from a config file. If we set a hard limit, somebody somewhere will have hard-limit + 1 tables :-) ?

Reply:
Ok, I buy that. I update the spec.